packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -1,18 +1,195 @@
 
 import NotificationsBox from './NotificationsBox.svelte';
+
+// Dashboard section configuration managed by dashboard page registry
+let dashboardSections = $state<ListOrganizerItem[]>([]);
+// eslint-disable-next-line svelte/no-unnecessary-state-wrap
+let dashboardOrdering = $state(new SvelteMap<string, number>());
+let isInitialized = false;
+let isLoading = false;
+
+let orderedSections = $derived(getOrderedDashboardSections());
+let updatedEntries = $derived(convertFromListOrganizerItems(orderedSections, dashboardPageRegistry.entries));
+
+// Filter and sort dashboard registry items based on LayoutEditor configuration
+let sortedDashboardRegistry = $derived(
+  orderedSections
+    .filter(section => section.enabled)
+    .map(section => dashboardPageRegistry.entries.find(item => item.id === section.id))
+    .filter((item): item is DashboardPageRegistryEntry => item?.component !== undefined),
+);
+
+// Initialize default dashboard configuration
+function getDefaultDashboardItems(): ListOrganizerItem[] {
+  return dashboardPageRegistry.entries.map(entry => ({
+    id: entry.id,
+    label: entry.id,
+    enabled: true,
+    originalOrder: entry.originalOrder,
+  }));
+}
+
+// Initialize dashboard configuration
+async function initializeDashboard(): Promise<void> {
+  if (isInitialized || isLoading) return;
+
+  isLoading = true;
+  try {
+    if (dashboardPageRegistry.entries.length > 0) {
+      const loadedItems = await loadDashboardConfiguration();
+      dashboardSections = loadedItems;
+    }
+    isInitialized = true;
+  } catch (error: unknown) {
+    console.error(`Failed to load dashboard configuration: ${error}`);
+    // Fallback to default configuration
+    dashboardSections = getDefaultDashboardItems();
+    isInitialized = true;
+  } finally {
+    isLoading = false;
+  }
+}
+
+// Load configuration from settings
+async function loadDashboardConfiguration(): Promise<ListOrganizerItem[]> {
+  if (!tablePersistence.storage) return getDefaultDashboardItems();
+
+  const loadedItems = await tablePersistence.storage.load('dashboard', defaultSectionNames);
+
+  if (loadedItems.length > 0) {
+    // Ensure loaded items have proper originalOrder from defaults if missing
+    const defaultItems = getDefaultDashboardItems();
+    const items = loadedItems.map((item: ListOrganizerItem) => ({
+      ...item,
+      originalOrder: item.originalOrder ?? defaultItems.find(d => d.id === item.id)?.originalOrder ?? 0,
+    }));
+
+    // Build ordering map from loaded items
+    // Check if items are in a different order than their original order
+    const isReordered = items.some((item: ListOrganizerItem, index: number) => item.originalOrder !== index);
+    if (isReordered) {
+      const ordering = new SvelteMap<string, number>();
+      items.forEach((item: ListOrganizerItem, index: number) => {
+        ordering.set(item.id, index);
+      });
+      dashboardOrdering = ordering;
```

Comment:
As per https://github.com/podman-desktop/podman-desktop/pull/13768/files#r2478242711 we should not re-assign




```suggestion
      items.forEach((item: ListOrganizerItem, index: number) => {
        dashboardOrdering.set(item.id, index);
      });
```

Reply:
Should be fixed